static/src/components/builder/utils.js

```diff
@@ -310,7 +310,7 @@ export default {
       {
         pybAnswer,
         choices: JSON.stringify(choices),
-        initialValue: JSON.stringify([initialValue]),
+        initialValue: JSON.stringify(initialValue || []),
```

Comment:
If we agree that initial-value will be a single item, I assume the only change needed to fix the blue line is to change this to:
```
initialValue: JSON.stringify([initialValue] if initialvValue else []),
```